collectors/bzimport/collectors.py

```diff
@@ -539,6 +539,10 @@ def collect(self, batch=None):
         msg += " Nothing new to fetch." if not flaw_ids else ""
         return msg
 
+    def __init__(self) -> None:
+        Collector.__init__(self)
+        JiraQuerier.__init__(self)
+
```

Comment:
I think the better solution is to change `JiraQuerier` to work like `BugzillaQuerier` (i.e. without an `__init__`, as a singleton)